src/app/frontend/chrome/component.js

```diff
@@ -50,11 +52,60 @@ export class ChromeController {
 
     /** @private {!kdUiRouter.$transitions} */
     this.transitions_ = $transitions;
+
+    /** @private {!angular.$resource} */
+    this.resource_ = $resource;
+
+    /** @private {!angular.$sce} */
+    this.sce_ = $sce;
+
+    /** @private {!backendApi.SystemBanner} */
+    this.systemBanner_;
   }
 
   /** @export */
   $onInit() {
     this.registerStateChangeListeners();
+    this.initSystemBanner_();
+  }
+
+  /**
+   * @private
+   */
+  initSystemBanner_() {
+    this.resource_('api/v1/systembanner').get((sb) => {
+      this.systemBanner_ = sb;
+    });
+  }
+
+  /**
+   * @export
+   * @return {boolean}
+   */
+  isSystemBannerVisible() {
+    return this.systemBanner_ !== undefined && this.systemBanner_.message.length > 0;
+  }
+
+  /**
+   * @export
+   * @return {string}
+   */
+  getSystemBannerClass() {
+    if (this.systemBanner_ && this.systemBanner_.severity) {
+      return `kd-system-banner-${this.systemBanner_.severity.toLowerCase()}`;
+    }
+    return '';
+  }
+
+  /**
+   * @export
+   * @return {*}
```

Comment:
{sting}? If possible

Reply:
Unfortunately not. `trustAsHtml` does not have precise return type and it can be `{Object|string|null}`

Reply:
Unfortunately not possible.